problem-solver/py/services/kindergartensModule/GetKindergartensByStreetAgent.py

```diff
@@ -0,0 +1,114 @@
+from termcolor import colored
+
+from common import ScAgent, ScEventParams, ScKeynodes
+from sc import *
+
+
+class GetKindergartensByStreetAgent(ScAgent):
+    def __init__(self, module):
+        super().__init__(module)
+        self.ctx = module.ctx
+        self.keynodes = ScKeynodes(self.ctx)
+        self.main_node = None
+
+    def RunImpl(self, evt: ScEventParams) -> ScResult:
+        self.main_node = evt.other_addr
+        status = ScResult.Ok
+
+        if self.module.ctx.HelperCheckEdge(
+                self.keynodes['action_get_kindergartens_by_street'],
+                self.main_node,
+                ScType.EdgeAccessConstPosPerm,
+        ):
+            try:
+                if self.main_node is None or not self.main_node.IsValid():
+                    raise Exception("The question node isn't valid.")
+                node = self.get_action_argument(self.main_node, 'rrel_1')
+                answerNode = self.ctx.CreateNode(ScType.NodeConstStruct)
+                self.add_nodes_to_answer(answerNode, [node])
+
+                streetIterator = self.ctx.Iterator5(
+                    node,
+                    ScType.EdgeDCommon,
+                    ScType.Unknown,
+                    ScType.EdgeAccessConstPosPerm,
+                    self.keynodes['nrel_street']
+                )
```

Comment:
если входной параметр у вас улица, то по этому итератору ничего не найдет
```suggestion
                streetIterator = self.ctx.Iterator5(
                    ScType.Unknown,
                    ScType.EdgeDCommon,
                    node,
                    ScType.EdgeAccessConstPosPerm,
                    self.keynodes['nrel_street']
                )
```

проверить эту ошибку во всех аентах